.circleci/config.yml

```diff
@@ -281,7 +281,7 @@ jobs:
       - *restore_rebar_cache
       - run:
           name: Build
-          command: ./rebar3 as test do release
```

Comment:
This does not seem to be the same thing?!

Reply:
Fixed.